test/integration_test/basic_test.go

```diff
@@ -104,17 +105,23 @@ var testStorageClusterBasicCases = []types.TestCase{
 		TestFunc:   InstallWithTelemetry,
 	},
 	{
-		TestName:        "InstallWithCSI",
+		TestName:        "BasicCsiRegression",
 		TestrailCaseIDs: []string{},
 		TestSpec: ci_utils.CreateStorageClusterTestSpecFunc(&corev1.StorageCluster{
-			ObjectMeta: meta.ObjectMeta{Name: "csi-test"},
-			Spec: corev1.StorageClusterSpec{
-				FeatureGates: map[string]string{"CSI": "true"},
-			},
+			ObjectMeta: meta.ObjectMeta{Name: "csi-regression-test"},
 		}),
 		ShouldSkip: func() bool { return false },
 		TestFunc:   BasicCsiRegression,
```

Comment:
nit: if no specific skip logic is needed we could delete this line, it will setup default function automatically.

Reply:
Removed